README.md

```diff
@@ -68,3 +68,11 @@ Unknown
 
 - Initial ODX Commander release
 
+## Development
+
```

Comment:
Nice documentation, but could we move all the development specific topics into an extra file? The Readme.md is visible as integrated help in VS Code and might confuse non developers

Reply:
✔️